apis/python/src/tiledbsoma/constants.py

```diff
@@ -3,3 +3,6 @@
 """
 
 SOMA_JOINID = "soma_joinid"
```

Comment:
probably worth adding `soma_data`, `soma_dim_` as these are in the spec, and widely used in the NDArray code

Reply:
at some point, these + soma_joinid should all  upstream into `somacore` as they are independent of the storage engine

Reply:
Works for me.  On the deferred items, are you tracking the backlog?  (I'm pretty sure you are, just confirming)